sql/parse/parse.go

```diff
@@ -373,6 +373,11 @@ func convertInsert(ctx *sql.Context, i *sqlparser.Insert) (sql.Node, error) {
 		return nil, ErrUnsupportedSyntax.New(i)
 	}
 
+	isReplace := false
+	if i.Action == "replace" {
```

Comment:
Should be a SQL constant for this, no?

Reply:
What do you mean? This is parsed as a `*sqlparser.Insert` statement, with the `i.Action` value set to `"replace"`. Unlike the other ones, this isn't parsed as its own, discrete action.